cli/cmd/plugin/unmanaged-cluster/cmd/list.go

```diff
@@ -34,6 +36,7 @@ var ListCmd = &cobra.Command{
 
 func init() {
 	ListCmd.Flags().BoolVar(&co.tty, "tty", true, "Specify whether terminal is tty. Set to false to disable styled output.")
+	ListCmd.Flags().StringVarP(&outputFormat, "output", "o", "", "Output format (yaml|json|table), default is table")
```

Comment:
use a semi colon here instead of `,` because consistency/convention.

ref: https://github.com/stmcginnis/community-edition/blob/fed519f1d01c2f1db4a977c1a41a6dbdd10e48b2/cli/cmd/plugin/unmanaged-cluster/cmd/create.go#L60

Reply:
while you're there, can you update --tty too.

i don't care what style we use, as long as it's consitent

Reply:
Done!